ckanext/switzerland/tests/test_ftp_storage_adapter.py

```diff
@@ -314,10 +313,10 @@ def test_fetch(self, MockFTP):
                 # fetch remote file
                 arg1, arg2 = ftph.fetch(filename, localpath=testfile)
         # tests
-        assert_equal(arg1, "RETR %s" % filename)
+        self.assertEqual(arg1, "RETR %s" % filename)
         log.debug(arg2)
-        # assert_equal(str(type(arg2)), "<type 'builtin_function_or_method'>")
-        assert_equal(str(arg2.__class__.__name__), "builtin_function_or_method")
+        # self.assertEqual(str(type(arg2)), "<type 'builtin_function_or_method'>")
```

Comment:
I assume this was already commented out, so I would get rid of it here, too, WDYT?

Reply:
oh yes, thank you for catching that :)